distrobox-init

```diff
@@ -1605,7 +1605,9 @@ if [ "${nvidia}" -eq 1 ]; then
 
 		if [ "${type}" = "link" ]; then
 			mkdir -p "$(dirname "${dest_file}")"
-			cp -d "${nvidia_lib}" "${dest_file}"
+			if [ "$(md5sum "${nvidia_lib}" | awk '{print $1}')" != "$(md5sum "${dest_file}" | awk '{print $1}')" ]; then
```

Comment:
Can we use `cut` instead of `awk`? just to be sure this work on minimal images :+1: